src/custom/state/claim/hooks/index.ts

```diff
@@ -244,7 +244,7 @@ export function useUserClaims(account: Account, optionalChainId?: SupportedChain
   const chainId = optionalChainId || connectedChain
 
   const [claimInfo, setClaimInfo] = useState<{ [account: string]: UserClaims | null }>({})
-  const [isLoading, setIsLoading] = useState(false)
+  const [isLoading, setIsLoading] = useState(true)
```

Comment:
I guess this one is independent change, could be done in another PR ideally. We can leave it here now its in this PR